css/index.css

```diff
@@ -0,0 +1,14 @@
+.row
+width 100%
+backround color: #FB9300
+
+.cards 
+width: 50px;
+height: 100px;
+border 5px, color: #F54748; 
```

Comment:
Reference the MDN here: 
https://developer.mozilla.org/en-US/docs/Web/CSS/border

It looks like you're trying to make a border 5px wide in that color code, but the syntax can be condensed into a single line here